src/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobWriter.cs

```diff
@@ -348,18 +348,18 @@ public void WriteDateTime(DateTime value)
         /// <remarks>
         /// References may be small (2B) or large (4B).
         /// </remarks>
-        public void WriteReference(uint reference, int size)
+        public void WriteReference(int reference, bool isSmall)
```

Comment:
I would put the large or small info somewhere more prominent than remarks, parameter doc for isSmall is probably better.